scrapers/wowgirlsblog.yml

```
@@ -0,0 +1,33 @@
+name: "WOW Girls Blog"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - wowgirlsblog.com
+
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h1
+      Date:
+        selector: //meta[@property="article:published_time"]/@content
+        postProcess:
+          - replace:
+            - regex: '([^ ]+)T.*'
+              with: $1
+      Details: //meta[@property="og:description"]/@content
+      Performers:
+        Name: //a[contains(@href,"actor")]/text()
+      Studio:
+        Name:
+          selector: //meta[@property="og:site_name"]/@content
+          postProcess:
+            - replace:
+              - regex: '([^ ]+) .*'
+                with: $1
+      Tags:
+        Name: //a[contains(@href,"category")]/@title
```

- studio can be fixed since it is wowgirls only?
- tags need to be cleaned
```suggestion
      Studio:
        Name:
          fixed: WowGirls
      Tags:
        Name:
          selector: //a[contains(@href,"category")]/@title
          postProcess:
            - replace:
                - regex: \s+Movies\s*$
                  with: ""
```

---

I would like to organize this in the tags-section of stash. But it depends...